### ISSUE TITLE
[Pods 2.8 Bug] Datepicker cuts off based on height of field group

### ISSUE THREAD

**Issue by jimtrue** [pods-framework/pods#5965]
## Pods 2.8 version

2.8.0-b-2

## Description

When using a Date Field, the DatePicker pop-up is only visible down to the field group and then cuts off.

## Testing instructions

1. Create a Date Field in the bottom of your Field Group
2. Attempt to set a date

## Expected behavior

The Datepicker should display normally, not cutoff.

## Screenshots / screencast

![datepicker-cutoff-error](https://user-images.githubusercontent.com/3052875/109984097-72257a80-7cd1-11eb-9a64-fbb48056e371.gif)

## Other super helpful information

<details>
<summary>WordPress Environment</summary>
  
```
### wp-core ###

version: 5.6.2
site_language: en_US
user_language: en_US
timezone: +00:00
permalink: /%postname%/
https_status: false
multisite: false
user_registration: 0
blog_public: 1
default_comment_status: open
environment_type: local
user_count: 1
dotorg_communication: true

### wp-paths-sizes ###

wordpress_path: /Users/jimtrue/Local Sites/podsbeta/app/public
wordpress_size: 43.22 MB (45317712 bytes)
uploads_path: /Users/jimtrue/Local Sites/podsbeta/app/public/wp-content/uploads
uploads_size: 0.00 B (0 bytes)
themes_path: /Users/jimtrue/Local Sites/podsbeta/app/public/wp-content/themes
themes_size: 6.28 MB (6583753 bytes)
plugins_path: /Users/jimtrue/Local Sites/podsbeta/app/public/wp-content/plugins
plugins_size: 114.46 MB (120017370 bytes)
database_size: 2.17 MB (2277376 bytes)
total_size: 166.13 MB (174196211 bytes)

### wp-active-theme ###

name: Twenty Twenty-One (twentytwentyone)
version: 1.1
author: the WordPress team
author_website: https://wordpress.org/
parent_theme: none
theme_features: core-block-patterns, automatic-feed-links, title-tag, post-formats, post-thumbnails, menus, html5, custom-logo, customize-selective-refresh-widgets, wp-block-styles, align-wide, editor-styles, editor-style, editor-font-sizes, custom-background, editor-color-palette, editor-gradient-presets, responsive-embeds, custom-line-height, experimental-link-color, custom-spacing, custom-units, widgets
theme_path: /Users/jimtrue/Local Sites/podsbeta/app/public/wp-content/themes/twentytwentyone
auto_update: Disabled

### wp-themes-inactive (2) ###

Twenty Nineteen: version: 1.9, author: the WordPress team, Auto-updates disabled
Twenty Twenty: version: 1.6, author: the WordPress team, Auto-updates disabled

### wp-plugins-active (1) ###

Pods - Custom Content Types and Fields: version: 2.8.0-b-2, author: Pods Framework Team, Auto-updates disabled

### wp-media ###

image_editor: WP_Image_Editor_GD
imagick_module_version: Not available
imagemagick_version: Not available
file_uploads: File uploads is turned off
post_max_size: 1000M
upload_max_filesize: 300M
max_effective_size: 300 MB
max_file_uploads: 20
gd_version: bundled (2.1.0 compatible)
ghostscript_version: not available

### wp-server ###

server_architecture: Darwin 20.3.0 x86_64
httpd_software: nginx/1.16.0
php_version: 7.3.5 64bit
php_sapi: fpm-fcgi
max_input_variables: 4000
time_limit: 1200
memory_limit: 256M
max_input_time: 600
upload_max_filesize: 300M
php_post_max_size: 1000M
curl_version: 7.54.0 LibreSSL/2.6.5
suhosin: false
imagick_availability: false
pretty_permalinks: true
htaccess_extra_rules: false

### wp-database ###

extension: mysqli
server_version: 8.0.16
client_version: mysqlnd 5.0.12-dev - 20150407 - $Id: 7cc7cc96e675f6d72e5cf0f267f48e167c2abb23 $

### wp-constants ###

WP_HOME: undefined
WP_SITEURL: undefined
WP_CONTENT_DIR: /Users/jimtrue/Local Sites/podsbeta/app/public/wp-content
WP_PLUGIN_DIR: /Users/jimtrue/Local Sites/podsbeta/app/public/wp-content/plugins
WP_MAX_MEMORY_LIMIT: 256M
WP_DEBUG: false
WP_DEBUG_DISPLAY: true
WP_DEBUG_LOG: false
SCRIPT_DEBUG: false
WP_CACHE: false
CONCATENATE_SCRIPTS: undefined
COMPRESS_SCRIPTS: undefined
COMPRESS_CSS: undefined
WP_LOCAL_DEV: undefined
DB_CHARSET: utf8
DB_COLLATE: undefined

### wp-filesystem ###

wordpress: writable
wp-content: writable
uploads: writable
plugins: writable
themes: writable

### pods ###

pods-server-software: nginx/1.16.0
pods-user-agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 11_2_2) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/89.0.4389.72 Safari/537.36
pods-session-save-path: undefined
pods-session-save-path-exists: No
pods-session-save-path-writable: No
pods-session-max-lifetime: 1440
pods-opcode-cache-apc: No
pods-opcode-cache-memcached: No
pods-opcode-cache-opcache: Yes
pods-opcode-cache-redis: No
pods-object-cache-apc: No
pods-object-cache-apcu: No
pods-object-cache-memcache: No
pods-object-cache-memcached: No
pods-object-cache-redis: No
pods-memory-current-usage: 5.281M
pods-memory-current-usage-real: 2.000M
pods-network-wide: No
pods-install-location: /Users/jimtrue/Local Sites/podsbeta/app/public/wp-content/plugins/pods/
pods-developer: No
pods-tableless-mode: No
pods-relationship-table-enabled: Yes
pods-light-mode: No
pods-strict: No
pods-allow-deprecated: Yes
pods-api-cache: Yes
pods-shortcode-allow-evaluate-tags: No

```

</details>

<details>
<summary>Pods Package Export</summary>

```json
{"meta":{"version":"2.8.0-b-2","build":1614870553},"pods":{"book":{"object_type":"pod","storage_type":"post_type","name":"book","id":5,"label":"Books","description":"","type":"post_type","storage":"meta","label_singular":"Book","public":"1","show_ui":"1","supports_title":"1","supports_editor":"1","_locale":"user","podType":{"name":"cpt","label":"Custom Post Type"},"storageType":{"name":"meta","label":"Meta"},"has_archive":"1","has_archive_slug":"books"}}}
```

</details>

**Comment by sc0ttkclark** [comment]
@zrothauser Here's another one for you to check in on

**Comment by sc0ttkclark** [comment]
Fixed via #6016
